### PR TITLE
Revert: use systemjs for javascript tests

### PR DESCRIPTION
**Pull request by OliverJAsh**
This is mostly a revert of #9103, but I had to manually downgrade the ES6 to ES5 and move any new tests/delete any old tests.

Revert "Merge pull request #9103 from guardian/systemjs-tests"

This reverts commit c51be021f31d15492bbf9b953ca81a723c6a95e5, reversing
changes made to 3803238b54be7e1f8b7d6e22f7958b9a400b6b34.

### CODE REVIEW

File: grunt-configs/jscs.js [hunk 1]
```diff
@@ -16,15 +16,29 @@ module.exports = function(grunt, options) {
                 ]
             }]
         },
+        'es6-tests': {
+            options: {
+                esnext: true
```
Comment (OliverJAsh) [inline grunt-configs/jscs.js]: Remove

File: grunt-configs/jscs.js [hunk 2]
```diff
@@ -16,15 +16,29 @@ module.exports = function(grunt, options) {
...
+                cwd: 'static/src/javascripts/tests',
+                src: [
+                    '**/*.js',
+                    '!components/**/*.js'
+                ]
+            }]
+        },
         test: {
             options: {
                 esnext: true
```
Comment (OliverJAsh) [inline grunt-configs/jscs.js]: Remove

**Comment by jbreckmckye** [comment]
How did you downgrade the tests - by hand?

**Comment by OliverJAsh** [comment]
The revert gave me back the AMD tests, so I just kept the dependency lists and just copied in the module contents from the ES6 tests. So there wasn't really much manual conversion, except for one or two arrow functions in the actual test contents.

**Comment by rich-nguyen** [comment]
:+1:

**Pull request merged**